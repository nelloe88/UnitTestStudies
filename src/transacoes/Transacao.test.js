import React from 'react';
import {render} from '@testing-library/react'
import Transacao from './Transacao'

describe('compoente de transacao do extratp',() => {
    it('O snapshot do compoente deve  permanecer sempre o mesmo', () => {

        const listaDeTransacoes = [
            { id: 1, tipo: 'saque', valor: '20.00', data: '08/09/2020' },
            { id: 2, tipo: 'deposito', valor: '25.00', data: '08/10/2020' },
            { id: 3, tipo: 'saque', valor: '30.00', data: '10/09/2020' }
          ]

          const {container} = render( <Transacao transacoes={listaDeTransacoes}   />)


          /*
        const {container} = render( <Transacao
            data="08/09/2020"
            tipo="saque"
            valor="20.00"

        />)*/

        expect(container.firstChild).toMatchSnapshot();
    })


})