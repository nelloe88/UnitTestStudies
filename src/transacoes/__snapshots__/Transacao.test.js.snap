// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compoente de transacao do extratp O snapshot do compoente deve  permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p />
  <p />
  <p>
    R$ 
    0
  </p>
</div>
`;
